pkg/appmanager/validateResources.go

```diff
@@ -363,3 +375,30 @@ func validateAppRootAnnotations(rsType int, entries map[string]string) {
 		}
 	}
 }
+
+// name:       checkAs3ConfigMap
+// arguments:  obj interface{} - ConfigMap Object
+// return val: bool - is it AS3 or not
+// description: This function validates configmap be AS3 specific or not
+
+func (appMgr *Manager) checkAs3ConfigMap(
+	obj interface{},
+) bool {
+	// check for metadata.labels has 'as3' and that 'as3' is set to 'true'
+	cm := obj.(*v1.ConfigMap)
+	labels := cm.ObjectMeta.Labels
+	log.Debugf("[as3_log] Parsing labels: %s ", labels)
+	if val, ok := labels["as3"]; ok {
+		log.Debugf("[as3_log] Found AS3 config map...")
+		if val == "true" {
+			return true
+		} else {
```

Comment:
https://github.com/F5Networks/k8s-bigip-ctlr/blob/71ffc366140fab51af0951ab6017a738739dfd8c/pkg/appmanager/validateResources.go#L395-L403
Let's replace this block with a single `return false`.

Reply:
log at line 400 is of less significance